src/components/TimeCardPage/TimeTableRow.tsx

```diff
@@ -7,6 +7,15 @@ import 'react-time-picker/dist/TimePicker.css';
 import 'react-clock/dist/Clock.css';
 import { Fragment } from 'react';
 
+import { Select } from '@chakra-ui/react'; 
+import {TIMEZONE} from 'src/constants'; 
+
+
+enum CellType {
+    Regular="Regular", 
+    PTO="PTO" 
+}
```

Comment:
For convention, can we put this in a `types.ts` file under the same directory? Easier to find for future reference/reuse